### PR TITLE
Add Settings screen

### PR DESCRIPTION
**Pull request by devinjameson**

### CODE REVIEW

File: src/Settings.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+import { Icons } from "./assets"
+import { Colors, Outlines, Sizing, Typography } from "./styles"
+
+interface SettingsProps {
+  componentId: string
+}
+
+const Settings: FunctionComponent<SettingsProps> = ({ componentId }) => {
+  const handleOnPressPrivacyPolicy = (): void => {
+    Linking.openURL("https://typeelephant.com")
```
Comment (johnschoeman) [inline src/Settings.tsx:26]: nbd but this might want to be an env variable.

File: src/Settings.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  componentId,
+}) => {
+  const colorScheme = useColorScheme()
+
+  const handleOnPressDismiss = (): void => {
+    Navigation.dismissModal(componentId)
+  }
+
+  const fill =
+    colorScheme === "light" ? Colors.constant.white : Colors.constant.black
```
Comment (johnschoeman) [inline src/Settings.tsx:96]: i wonder if this wants to be a style function. something like:
```
const fill = Colors.iconFill(colorScheme)
```

**Review by johnschoeman — APPROVED** [review]
LGTM!

File: src/Settings.tsx (hunk 2 above)
Comment (devinjameson) [reply src/Settings.tsx:96]: What's your reasoning here? What if these colors want to change for each icon?

**Review by johnschoeman — APPROVED** [review]

**Pull request merged**